### PR TITLE
[LTC] Refactor computation_client/util.h

### PR DESCRIPTION
**Pull request by alanwaketan**
Summary:
This commit refactors computation_client/util.h:
1) Consolidates some of the utils with c10/std equivalent.
2) Removes unused ones.

Test Plan:
lazy_tensor_core/test/cpp/build/test_ptltc

### CODE REVIEW

File: torch/csrc/lazy/core/hash.h [hunk 1]
```diff
@@ -181,6 +181,11 @@ static inline hash_t Hash(const hash_t& value) {
   return value;
 }
 
+template <typename T>
```
Comment (wconstab) [inline torch/csrc/lazy/core/hash.h:184]: can you also make a PR to land this part to master?

**Review by wconstab — APPROVED** [review]
Looks great, thanks @alanwaketan!

File: torch/csrc/lazy/core/hash.h (hunk 1 above)
Comment (alanwaketan) [reply torch/csrc/lazy/core/hash.h:184]: Yea, for sure. Need more time to work on the upstream counterpart.

**Comment by alanwaketan** [comment]
> Looks great, thanks @alanwaketan!

Thanks, Will.

**Pull request merged**